### PR TITLE
Rename annotations for the C/C++ compilers and linker to `cflags` and `ldflags`

### PR DESCRIPTION
**Pull request by xymus**
The new name is shorter, easier to remember and more standard. I may soon extend these annotations to provide information per platform, mainly to provide ldflags for Android only.

### CODE REVIEW

**Comment by jpages** [comment]
+1

**Comment by xymus** [comment]
ok to test

File: lib/realtime.nit [hunk 1]
```diff
@@ -11,7 +11,7 @@
 # another product.
 
 # Provides the Clock utility class to keep time of real time flow
-module realtime is c_linker_option("-lrt")
+module realtime is ldflags("-lrt")
```
Comment (privat) [inline lib/realtime.nit]: reste des parenthèses

**Comment by privat** [comment]
+1

**Comment by xymus** [comment]
@privat , tu vas aimer la suite à cette PR. Je vais proposer ma première annotation d'annotation.

**Comment by Morriar** [comment]
+1

**Pull request merged**